### DISCUSSION TITLE
Allow to run TDM with different configurations

### DISCUSSION THREAD

**Discussion by barbudor** [jziolkowski/tdm#211] [Ideas]
@jziolkowski 

As of today I'm running 3 different Tasmota/MQTT environment:
- 1 home local
- 1 remote reachable through a VPN
- 1 remote using a cloud-base broker

To use TDM with those 3 environment, I'm using scripts to move the `TDM.cfg`, `devices.cfg` and `TDM.log` between the default TDM folder and a saving folder (I could have used links to...)

However a command line option to specify the configuration folder on the command line would be a convenient way of easily switching between the configuration.

**Comment by jziolkowski** [comment]
I've been thinking about this as well, because now I manage 3 locations as well :)

So the ideas are:

- multi-broker support (so each list of devices is displayed in its separate tab)
- command-line parameter with path to settings file

The options are not mutually exclusive, of course

**Comment by jziolkowski** [comment]
custom locations for config/logging are WIP in #215

**Comment by jziolkowski** [comment]
done, released today
